src/Polly.Strategies/Retry/RetryStrategyOptions.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Polly.Retry;
+public class RetryStrategyOptions : ResilienceStrategyOptions
+{
+    public Predicates<ShouldRetryArguments> ShouldRetry { get; set; } = new();
+
+    public Events<RetryActionArguments> OnRetry { get; set; } = new();
+
+    public int RetryCount { get; set; } = 3;
+
+    public Func<int, TimeSpan> RetryDelayGenerator { get; set; } = _ => TimeSpan.FromSeconds(3);
```

Comment:
I'm guessing these are our current defaults in v7?

Reply:
Not really, just some random values I picked. we can fine-tune these in real PRs.